pkg/localmetrics/localmetrics.go

```diff
@@ -89,16 +93,28 @@ var (
 
 // UpdateAWSMetrics updates the total AWS Accounts metric every N hours
 func UpdateAWSMetrics(kubeClient kubeclientpkg.Client, hour int) {
-	awsClient, _ := awsclient.GetAWSClient(kubeClient, awsclient.NewAwsClientInput{
+	metricLogger := log.WithValues("Namespace", "aws-account-operator-operator")
+
+	awsClient, err := awsclient.GetAWSClient(kubeClient, awsclient.NewAwsClientInput{
 		SecretName: account.AwsSecretName,
 		NameSpace:  awsv1alpha1.AccountCrNamespace,
 		AwsRegion:  "us-east-1",
 	})
 
+	if err != nil {
+		metricLogger.Error(err, "Failed to get awsClient")
+		return
+	}
+
 	d := time.Duration(hour) * time.Minute
 	for range time.Tick(d) {
-		accountTotal, _ := account.TotalAwsAccounts(awsClient)
-		MetricTotalAWSAccounts.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(accountTotal))
+		accountTotal, err := account.TotalAwsAccounts(awsClient)
+
+		if err != nil {
+			metricLogger.Error(err, fmt.Sprintf("Failed to get total number of AWS accounts: %s", err))
```

Comment:
Logging here is good, but one has to actually look at the logs to know this is failing. Maybe we set the metric here to `-1` and alert on it if its `-1` for some number of hours. Thoughts?

Reply:
Thats the problem we have right now though right it gets set to 0 and our graphs get screwed?

Reply:
Yeah I guess this would screw up history. Ignore this then